core/src/main/java/com/amplifyframework/devmenu/DeveloperMenu.java

```diff
@@ -160,12 +160,11 @@ public String getFilteredLogs(String searchText) {
 
     /**
      * Creates and returns the issue body containing the issue description,
-     * environment information, device information, and (optionally) logs.
+     * environment information, and device information.
      * @param description the description of the issue
-     * @param includeLogs whether to include logs in the issue body
      * @return the issue body
      */
-    public String createIssueBody(String description, boolean includeLogs) {
+    public String createIssueBody(String description) {
```

Comment:
Rather than remove this good work you already did, I would just modify the code which calls the function to always send false for now with a ToDo comment that for now logs aren't being included but if that changes later they can be sent.

Reply:
That seems good to me! I advised Erica to remove code versus comment out, but I think the solution above is reasonable and ensures we can easily pick this back up.

Reply:
Yeah it makes sense to me to remove the UI code but as far as this method goes it's optional, additional functionality so felt like it made sense to make an exception here.